re-former/app/views/users/new.html.erb

```diff
@@ -0,0 +1,50 @@
+<h1>New User</h1>
+
+<!--
+<form action="/new" accept-charset="UTF-8" method="POST">
+<input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+
+  <label for="username">User Name:</label>
+  <input type="text" name="user[username]" id="username">
+  <br><br>
+  <label for="email">Email:</label>
+  <input type="email" name="user[email]" id="email">
+  <br><br>
+  <label for="password">Password:</label>
+  <input type="password" name="user[password]" id="password">
+  <br><br>
+  <button type="submit" name="submit">Save</button>
+</form>
+-->
+<!--
+<%= form_with(model: @user, url: "/new", method: "POST") do %>
+  <%= label_tag(:username, "User Name:") %>
+  <%= text_field(:user,:username) %>
+
+  <%= label_tag(:email, "Email:") %>
+  <%= email_field(:user,:email) %>
+
+  <%= label_tag(:password, "Password:") %>
+  <%= password_field(:user,:password) %>
+
+  <%= submit_tag("Save") %>
+<% end %>
+-->
+
+<%= form_for( @user, url: "/users/new")  do |f| %>
```

Comment:
You don't need to specify here the `url` option, with the instance variable is enough.